### ISSUE TITLE
[Feature Request] Option to remove results that would be larger

### ISSUE THREAD

**Issue by Stanzilla** [xanderfrangos/crushee#6]
Heya, it would be nice to get an option/button in the arrow menu for "Remove All" to remove all the results that would result in a larger image.

**Comment by xanderfrangos** [comment]
Good idea. There's current an option in the "Edit" menu called "Remove unoptimized files" that does what you're looking for, but I should add it to the Remove menu as well.

**Comment by positlabs** [comment]
I don't see any use-case where anyone would actually want to save the files that are larger. I would prefer to have those results blocked from saving by default, but have the option to enable that behavior. If there are some analytics added, you could see if anyone actually uses the enlarger option, or if it can be removed.

**Comment by positlabs** [comment]
For comparison, imageOptim does this by default. If an image size isn't smaller than the original, it just shows a red x on it to signify there was no size reduction
